tox.ini

```diff
@@ -41,37 +40,41 @@ commands = isort --profile black --check --diff .
+    pylint src/probnum/filtsmooth --disable="no-member,arguments-differ,arguments-renamed,too-many-arguments,too-many-locals,too-few-public-methods,protected-access,unused-variable,unused-argument,no-else-return,no-self-use,duplicate-code,line-too-long,missing-module-docstring,missing-class-docstring,missing-function-docstring" --jobs=0
+    pylint src/probnum/kernels --disable="too-few-public-methods,no-else-return,duplicate-code" --jobs=0
+    pylint src/probnum/linalg --disable="no-member,abstract-method,arguments-differ,redefined-builtin,too-many-instance-attributes,too-many-arguments,too-many-locals,too-many-lines,too-many-statements,too-many-branches,too-complex,too-few-public-methods,protected-access,unused-argument,attribute-defined-outside-init,no-else-return,no-else-raise,no-self-use,duplicate-code,line-too-long,missing-module-docstring" --jobs=0
+    pylint src/probnum/linops --disable="too-many-instance-attributes,too-many-arguments,too-many-locals,protected-access,no-else-return,no-else-raise,line-too-long,missing-class-docstring,missing-function-docstring" --jobs=0
+    pylint src/probnum/problems --disable="too-many-arguments,too-many-locals,unused-variable,unused-argument,consider-using-from-import,duplicate-code,line-too-long,missing-module-docstring,missing-function-docstring" --jobs=0
+    pylint src/probnum/quad --disable="function-redefined,too-many-arguments,line-too-long,missing-module-docstring,missing-function-docstring" --jobs=0
+    pylint src/probnum/randprocs --disable="arguments-differ,arguments-renamed,too-many-instance-attributes,too-many-arguments,too-many-locals,protected-access,unused-argument,no-else-return,duplicate-code,line-too-long,missing-module-docstring,missing-function-docstring" --jobs=0
+    pylint src/probnum/randvars --disable="too-many-arguments,too-many-locals,too-many-branches,too-few-public-methods,protected-access,unused-argument,no-else-return,duplicate-code,line-too-long,missing-function-docstring" --jobs=0
+    pylint src/probnum/utils --disable="no-else-return,line-too-long" --jobs=0
+    # Benchmark and Test Code Linting Pass
+    # pylint benchmarks --disable="unused-argument,attribute-defined-outside-init,missing-function-docstring" --jobs=0 # not a work in progress, but final
+    pylint benchmarks --disable="unused-argument,attribute-defined-outside-init,no-else-return,no-self-use,consider-using-from-import,line-too-long,missing-module-docstring,missing-class-docstring,missing-function-docstring" --jobs=0
+    pylint tests --disable="arguments-differ,redefined-outer-name,too-many-instance-attributes,too-many-arguments,too-many-locals,too-few-public-methods,protected-access,unnecessary-pass,unused-variable,unused-argument,unused-private-member,attribute-defined-outside-init,no-else-return,no-self-use,consider-using-from-import,duplicate-code,line-too-long,missing-module-docstring,missing-class-docstring,missing-function-docstring" --jobs=0
```

Comment:
holy shit :D are these all previous global ignores, just merged here? Or have you checked whether some of these are necessary? (I think moving is enough for this PR, but was curious).

Reply:
I manually checked all of them so this should be the minimal list for each package